frontend/appflowy_flutter/assets/translations/hi.json

```diff
@@ -0,0 +1,441 @@
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है",
+    "alreadyHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "repeatPasswordHint": "रिपीट पासवर्ड"
+  },
+  "signIn": {
+    "loginTitle": "लॉगिन करे @:appName",
+    "loginButtonText": "लॉगिन",
+    "loginAsGuestButtonText": "शुरू करे",
+    "buttonText": "साईन करे",
+    "forgotPassword": "क्या आप पासवर्ड भूल गए ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "dontHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है"
+  },
+  "workspace": {
+    "create": "काम करने की जगह बनाये",
+    "hint": "काम करने की जगह",
+    "notFoundError": "काम करने की जगह नहीं मिली"
+  },
+  "shareAction": {
+    "buttonText": "शेयर करे",
+    "workInProgress": "जल्द आ रहा है",
+    "markdown": "मार्क कीजिये",
+    "copyLink": "लिंक कॉपी करे"
+  },
+  "moreAction": {
+    "small": "छोटा",
+    "medium": "बड़ा",
+    "large": "बहोत बड़ा",
+    "fontSize": "अक्षर साइज",
+    "import": "बहार से लाये",
+    "moreOptions": "और विकल्प"
+  },
+  "disclosureAction": {
+    "rename": "नाम बदले",
+    "delete": "इसे मिटायें",
+    "duplicate": "डुप्लिकेट"
+  },
+  "blankPageTitle": "खली पेज",
+  "newPageText": "नया पेज",
+  "trash": {
+    "text": "कूड़ा",
+    "restoreAll": "सभी बहाल करो",
+    "deleteAll": "सभी हटा दो",
+    "pageHeader": {
+      "fileName": "फ़ाइल का नाम",
+      "lastModified": "अंतिम बार संशोधित",
+      "created": "बन गया"
+    }
+  },
+  "deletePagePrompt": {
+    "text": "यह पृष्ठ कूड़े दान में है",
+    "restore": "पृष्ठ पुनर्स्थापित करें",
+    "deletePermanent": "हमेशा के लिए हटाए"
+  },
+  "dialogCreatePageNameHint": "पृष्ठ का नाम",
+  "questionBubble": {
+    "shortcuts": "शॉर्टकट",
+    "whatsNew": "नया क्या है ?",
+    "help": "मदद समर्थन",
+    "markdown": "मार्क कीजिये",
+    "debug": {
+      "name": "डिबग जानकारी",
+      "success": "डिबग जानकारी क्लिपबोर्ड पर कॉपी की गई!",
+      "fail": "डिबग जानकारी को क्लिपबोर्ड पर कॉपी करने में असमर्थ"
+    }
+  },
+  "menuAppHeader": {
+    "addPageTooltip": "जल्दी से अंदर एक पन्ना जोड़ें",
+    "defaultNewPageName": "शीर्षकहीन",
+    "renameDialog": "नाम बदले"
+  },
+  "toolbar": {
+    "undo": "अनडू करे",
+    "redo": "रीडू करे",
+    "bold": "बोल्ड बनाये",
+    "italic": "इटालिक करे ",
+    "underline": "अंडरलाइन करे",
+    "strike": "स्ट्रिकथ्रौघ",
+    "numList": "नंबर लिस्ट",
+    "bulletList": "बुलेट लिस्ट",
+    "checkList": "चेक लिस्ट",
+    "inlineCode": "इनलाइन कोड",
+    "quote": "क्वोट ब्लॉक",
+    "header": "हैडर",
+    "highlight": "हाईलाइट",
+    "color": "कलर"
+  },
+  "tooltip": {
+    "lightMode": "लाइट मोड",
+    "darkMode": "डार्क मोड ",
+    "openAsPage": "पृष्ट के जैसे खोले",
```

Comment:
Above we are using `पेज` for Page so please maintain that consistency throughout the keys.